libs/form-renderer/src/FormRenderer.tsx

```diff
@@ -88,11 +93,12 @@ export type Props = {
 }
 
 export const FormRenderer = ({ formComponents, action, submitButtonText }: Props) => {
+  const hasOneFormComponent = formComponents.length === 1
```

Comment:
why is this defined here and in a different way in Additional forms? Can't you just pass a prop with default value and set it when there are more then one form components?